postgres/server/init.sls

```diff
@@ -70,6 +70,9 @@ postgresql-config-dir:
     - makedirs: True
     - require:
       - cmd: postgresql-cluster-prepared
+    {%- if postgres.conf_dir == postgres.data_dir %}
+    - dir_mode: 0700
```

Comment:
This is a good change, and the configuration directory should definitely have to be secured even if does not match the data directory. No need for `if` statement. Just put it above of the `require` items as a matter of consistent style.

Reply:
I was reading PG documentation regarding this (due to error in logs).  I could not find any instruction that conf_dir needs to be secured.  I understand you see no issues using 0700 for conf_dir so agree here.

Reply:
I have implemented this. Because it's allowed to have cond_dir and data_dir the same in PG implementations, and 0700 was already requirement, then 0700 applies to conf_dir too.  You are correct therefore - the if should be removed.